extension/externalsubscriber/option.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package externalsubscriber
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/event"
+	"github.com/ava-labs/hypersdk/vm"
+)
+
+const (
+	Namespace = "externalSubscriber"
+)
+
+type Config struct {
+	Enabled       bool   `json:"enabled"`
+	ServerAddress string `json:"serverAddress"`
+}
+
+func NewConfig() Config {
+	return Config{
+		Enabled: false,
+	}
+}
```

Comment:
nit: `NewDefaultConfig` and we can just return an empty struct since this defaults to false